cnf-certification-test/lifecycle/suite.go

```diff
@@ -484,3 +490,51 @@ func testPodPersistentVolumeReclaimPolicy(env *provider.TestEnvironment) {
 		ginkgo.Fail(errMsg)
 	}
 }
+
+func testCPUIsolation(env *provider.TestEnvironment) {
+	ginkgo.By("Testing pods for CPU isolation requirements")
+
+	// Individual requirements we are looking for:
+	// - Resource Requests and Limits must be provided and identical. This determines whether or not we run the rest of the tests.
+	// - CPU Requests and Limits must be in the form of whole units
+	// - 'runtimeClassName' must be specified
+	// - Annotations must be provided disabling CPU and IRQ load-balancing
+
+	podsMissingIsolationRequirements := make(map[string]bool)
+
+	for _, put := range env.Pods {
+		it := isolation.NewCPUIsolationTester(put)
+
+		// Pods that have identical resources, are considered CPU isolated.
+		if it.LoadBalancingDisabled() {
```

Comment:
Shouldn't it be it.AreResourcesIdentical() here ?

Reply:
Nope, the DOORS requirement says that it's the annotations that determine the isolation status.

Reply:
I added `put.IsCPUIsolationEnabled() && !put.IsCPUIsolationCompliant()` as the check.